bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/CTabFolder2Listener.java

```diff
@@ -115,6 +115,34 @@ public interface CTabFolder2Listener extends SWTEventListener {
  */
 public void showList(CTabFolderEvent event);
 
+/**
+ * Sent when the tab items count changes
+ * 
+ * @param event from observed tab folder
+ * @since 3.124
+ */
+public default void itemsCount(CTabFolderEvent event) {
```

Comment:
I would rather call it `itemsCountChanged` so that it doesn't sound like a verb.

Reply:
Funny enough, that was my first name. But looking at other events, I see they all are verbs: `maximize`, `close`, etc (instead of `maximized`, `closed` etc). To be consistent, I would keep the "verb" form here, because it's already there.

Reply:
OK